assets/cubyz/recipes/wood_recipes

```diff
@@ -1,20 +1,75 @@
-L = cubyz:oak_log
-T = cubyz:oak_top
-P = cubyz:oak_planks
-C = cubyz:coal
+cubyz:birch_planks
+result 2*cubyz:stick
 
-L
-result 4*P
+cubyz:oak_planks
+result 2*cubyz:stick
 
-T
-result 3*P
+cubyz:mahogany_planks
+result 2*cubyz:stick
 
-4*P
+cubyz:pine_planks
+result 2*cubyz:stick
+
+4*cubyz:birch_planks
+result cubyz:workbench
+
+4*cubyz:oak_planks
+result cubyz:workbench
+
+4*cubyz:mahogany_planks
+result cubyz:workbench
+
+4*cubyz:pine_planks
 result cubyz:workbench
 
-C
-P
+cubyz:birch_log
+result 4*cubyz:birch_planks
+
+cubyz:birch_top
+result 3*cubyz:birch_planks
+
+cubyz:oak_log
+result 4*cubyz:oak_planks
+
+cubyz:oak_top
+result 3*cubyz:oak_planks
+
+cubyz:mahogany_log
+result 4*cubyz:mahogany_planks
+
+cubyz:mahogany_top
+result 3*cubyz:mahogany_planks
+
+cubyz:pine_log
+result 4*cubyz:pine_planks
+
+cubyz:pine_top
+result 3*cubyz:pine_planks
+
+cubyz:birch_planks
+result 2*cubyz:birch_fence
+
+cubyz:oak_planks
+result 2*cubyz:oak_fence
+
+cubyz:mahogany_planks
+result 2*cubyz:mahogany_fence
+
+cubyz:pine_planks
+result 2*cubyz:pine_fence
+
+cubyz:coal
+cubyz:birch_planks
```

Comment:
Maybe use a stick for torches, now that it has a recipe again?

Reply:
Yeah, that's a good idea. This would also allow having different tool properties for the different wood types.